docs/source/use_cases/run_pipelines_in_the_cloud/index.rst

```diff
@@ -0,0 +1,95 @@
+==========================================
+Run workflows and common tasks in parallel
+==========================================
+
+.. comment: begin: goto-read-the-docs
+
+.. container:: visible-only-on-github
+
+   +-----------------------------------------------------------------------------------+
+   | **The properly rendered version of this document can be found at Read The Docs.** |
+   |                                                                                   |
+   | **If you are reading this on github, you should instead click** `here`__.         |
+   +-----------------------------------------------------------------------------------+
+
+.. _RenderedVersion: http://googlegenomics.readthedocs.org/en/latest/use_cases/run_pipelines_in_the_cloud/index.html
+
+__ RenderedVersion_
+
+.. comment: end: goto-read-the-docs
+
+Do you have a task that you need to run independently over dozens,
+hundreds, or thousands of files in Google Cloud Storage? The
+`Google Genomics Pipelines API`_ provides an easy way to launch
+and monitor tasks running in the cloud.
+
+  +---------------------------------------------------------------------+
+  | Alpha                                                               |
+  +---------------------------------------------------------------------+
+  | This is an Alpha release of Google Genomics API. This feature might |
+  | be changed in backward-incompatible ways and is not recommended for |
+  | production use. It is not subject to any SLA or deprecation policy. |
+  +---------------------------------------------------------------------+
+
+Overview
+--------
+
+A "pipeline" in its simplest form is simply a task consisting of:
+
+  * Path(s) of input files to read from Cloud Storage
+  * Path(s) of output files/directories to write to Cloud Storage
+  * A Docker_ image to run
+  * A command to run in the docker image
+  * Cloud resources to use (number of CPUs, amount of memory, disk size and type)
+
+The Pipelines API will:
+
+  #. Create a Compute Engine virtual machine
+  #. Download the docker image
+  #. Download the input files
+  #. Run the docker command
+  #. Upload the output files
+  #. Destroy the Compute Engine virtual machine
+
+Log files are uploaded periodically to Cloud Storage.
+
+Alternatives
+------------
+
+For many cases, the Pipelines API has an advantage over fixed clusters
+in that Compute Engine resources (virtual machines and disks) are
+allocated only for the lifetime of the running pipeline and are then
```

Comment:
nit, probably adding a comma after pipeline might help, as it might become a run-on sentence.  It is good this way too, but it might be read several times :)